data.sql

```diff
@@ -6,3 +6,69 @@ VALUES
   ('Gabumon', '2018-11-15', 2, true, 8),
   ('Pikachu', '2021-01-07', 1, false, 15.04),
   ('Devimon', '2017-05-12', 5, true, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Charmander', '2020-02-08', 0, false, -11, 'Fire'),
+       ('Plantmon', '2021-11-15', 2, true, -5.7, 'Grass'),
+       ('Squirtle', '1993-04-02', 3, false, -12.13, 'Water'),
+       ('Angemon', '2005-06-12', 1, true, -45, 'Angel'),
+       ('Boarmon', '2005-06-07', 7, true, 20.4, 'Mammal'),
+       ('Blossom', '1998-10-13', 3, true, 17, 'Flower'),
+       ('Ditto', '2022-05-14', 4, true, 22, 'Normal');
+
+BEGIN;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction%20update%20the%20animals%20table%20by%20setting%20the%20species%20column%20to%20unspecified), Kindly add a transaction that updates the animals' table by setting the `species `column to `unspecified`. Verify that the change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.